Engine/source/gui/editor/guiMenuBar.h

```diff
@@ -161,21 +163,25 @@ class GuiMenuBar : public GuiTickCtrl //  Was: GuiControl
 
 	// internal menu handling functions
 	// these are used by the script manipulation functions to add/remove/change menu items
-
+   static Menu* sCreateMenu(const char *menuText, U32 menuId);
```

Comment:
Why is this not `createMenu`? I assume `s` short for static, of which doesn't follow the rest of the static method naming conventions.